### DISCUSSION TITLE
Printers

### DISCUSSION THREAD

**Discussion by roberkane212** [AlexGustafsson/homebridge-wol#147] [Questions]
Does anyone know how to use this with printers? I’d love to be able to turn on and off my cannon printer

**Comment by AlexGustafsson** [comment]
From my experience printers do not feature Wake on LAN. So turning it on might be hard. They usually do however feature some protocols for communication. If it doesn’t have a HTTP server running from which you can maintain it, it might be running a telnet server. You’d most likely need to do some digging on Google. Scanning the printer using the program nmap might help you discover what communication channels it has.

What’s the full model name of your printer?

**Comment by roberkane212** [comment]
@AlexGustafsson ahhh ok thanks I figured it must be something since I didn’t see a config with one. I did some research all I see if print and scan commands no on off commands. My printer is a cannon TR4500 series

**Comment by roberkane212** [comment]
Exact model pixma TR4520

**Comment by AlexGustafsson** [comment]
I can't seem to find anything that may enable this. If you've found anything or end up finding anything, feel free to post about it here.

**Comment by AlexGustafsson** [comment]
I'm transferring this into a discussion as it's not a software issue related to the plugin as for now.
